dist/tcti-tabrmd.pc.in

```diff
@@ -1,7 +1,7 @@
 Name: tcti-tabrmd
 Description: TCTI library for communicating with the TPM2 access broker / resource manager daemon (tabrmd).
-URL: https://github.com/01org/??
+URL: https://github.com/01org/tpm2-abrmd
 Version: @VERSION@
-Requires: sapi
+Requires: sapi glib-2.0 gio-2.0
```

Comment:
Requires: sapi glib2.0-dev

The dev version pulls in the bits needed. gio-2.0 isn't its own thing as far as packaging goes.

```
$ apt-file search gio/gio.h
libglib2.0-dev: /usr/include/glib-2.0/gio/gio.h
```